### ISSUE TITLE
import Statement throws Syntax Error even after Node Update

### ISSUE THREAD

**Issue by telion2** [interactionlab/NLATool#32]
Node Version: 9.0.0
npm Version: 5.5.1
Error Message: 
import CoreNLP, { Properties, Pipeline, ConnectorServer } from 'corenlp';
^^^^^^

SyntaxError: Unexpected token import
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:152:10)
    at Module._compile (module.js:605:28)
    at Object.Module._extensions..js (module.js:652:10)
    at Module.load (module.js:560:32)
    at tryModuleLoad (module.js:503:12)
    at Function.Module._load (module.js:495:3)
    at Module.require (module.js:585:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Medieninformaitk\Medieninformatik\ProjektTextBezuge\Repo\NLA-Projekt\routes\loadtext.js:25:17)

The error occurs in /modules/corenlp line 20
I changed Syntax Highlighting of Webstorm to ECMA Script 6 -> It wont show the syntax Error of the import statement in the IDE. 
I thought that if node is updated it supports es6 directly. Is that wrong or do I have to install es6 too:
(https://www.npmjs.com/package/es6) or (https://www.npmjs.com/package/node-es6)
I did some reading and a lot of times the transpiler Babel is mentioned that is needed to run such scripts. 
However that was always in a context of the year 2016 so I thought that has nothing to do with our app since I thought node supports ES6. 
This website (https://nodejs.org/en/docs/es6/) also tells me that node supports es6 and the command node -p process.versions.v8 had a result of 6.2.414.32-node.8 which lead me to this website: (https://github.com/nodejs/node/tree/master/deps/v8) that also told me that node is operating with ES6 (If I understood it correctly ECMA-262 is basically ES6). 
So basically I have no idea where to look anymore. Is there something I missed?
